File: src/BibliotecaCorporativa/backend/BibCorp.Domain/Models/Usuarios/Usuario.cs
using BibCorp.Domain.Models.Emprestimos;
using Microsoft.AspNetCore.Identity;

namespace BibCorp.Domain.Models.Usuarios
{
  public class Usuario : IdentityUser<int>
  {
    public string Nome { get; set; }
    public string Localizacao { get; set; }
    public string FotoURL { get; set; }
    public IEnumerable<Emprestimo> Emprestimos { get; set; }
    public IEnumerable<UsuarioPapel> UsuariosPapeis { get; set; }
  }
}

File: src/BibliotecaCorporativa/frontend/BibCorp-app/src/app/models/Usuarios/Usuario.ts
import { type Emprestimo } from '../Emprestimos/Emprestimo'

export class Usuario{
    id: number
    nome: string
    localizacao: string
    email: string
    password: string
    userName:string
    phoneNumber:string
    Emprestimos: Emprestimo[]
    token: string
}
